doc/usage.rst

```diff
@@ -54,13 +54,17 @@ Documentation items can be defined using the *item* directive, specifying:
         :validated_by: ITEST-MY_FIRST_INTEGRATION_TEST
         :ext_polarion_reference: project_x:workitem_y
         :nocaptions:
+        :hidelinks: validated_by
 
         According to the Polarion reference, the software **shall** implement my first requirement.
 
 Attributes can be added to the item, using the configured attribute keys in :ref:`traceability_default_config`
 (e.g. *value* in the above example). The content of the attribute is treated as a single string and should
 match_ the regular expression in configuration.
 
+The *hidelinks* is a space-seperated list of relation types. The relations to other items will not be shown for the
+mentioned relation types below the item. The links will still be used and displayed in traceability matrixes and other directives.
+
```

Comment:
Please merge this paragraph with the paragraph on line 80-81 like so:

```
:hidetype: *optional*, *multiple arguments (space-separated)*

    A space-separated list of relation types. By default, the rendered item definition displays all relations to other items
    in a list below the item name (id). The relation types given to this option will be omitted from this list.
    It does not affect how the item is rendered in other directives, e.g. item-matrix.

:nocaptions: *optional*, *flag*

    Hides the caption in the rendered item definition.
```
It is my intention to document all options of all directives in this way. Do you like it?

Reply:
yep looks good, and uniformity is even better ;-)